egg-share-x-upload-server.json

```diff
@@ -1,16 +1,16 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
     "exported_at": "2021-05-18T22:02:56-04:00",
     "name": "ShareX Upload Server",
-    "author": "danielpd93@gmail.com",
+    "author": "dan@danbot.host",
     "description": "ShareX Egg, Made by DBH",
     "features": null,
     "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-12"
```

Comment:
Are you sure ShareX's server supports the latest version of Node JS?

Reply:
I think it would be safer to use NodeJS 16 for the ShareX server

Reply:
I also think it'd be safer to use NodeJS 16, if those changes are made then I can consider this pull request and merge it.

Reply:
> I also think it'd be safer to use NodeJS 16, if those changes are made then I can consider this pull request and merge it.

✔ Done